guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,34 +1,6 @@
 [id="Upgrading_the_External_Database_{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
-
-.Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
-
-.Procedure
-. Create a backup.
-. Restore the backup on the new server.
-ifdef::katello,satellite,orcharhino[]
-. Correct the permissions on the `evr` extension:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# runuser -l postgres -c \
-"psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
-----
-endif::[]
```

Comment:
This part is really important and specific because it's a Katello extension, not part of standard PostgreSQL. You **must** do this after upgrading to PostgreSQL 13 or users will run into problems in the migration to drop it.